### ISSUE TITLE
Fix 4 pytest failed error in `src/regolith/tests/test_builders.py`

### ISSUE THREAD

**Issue by bobleesj** [regro/regolith#1186]
### Problem

We have `AssertionError` shown below:

<img width="1184" alt="Screenshot 2024-11-08 at 12 33 09 AM" src="https://github.com/user-attachments/assets/c52ddf73-a6ed-4b5e-b355-ca06c161af3d">

How to reproduce: @tinatn29 

```python
conda create -n regolith_env python=3.11
conda activate regolith_env
pip install -e .
pip install -r requirements/test.txt
pytest
```

### Proposed solution

I haven't looked into the code yet. But it appears to be comparing two strings. 

Once it is resolved, create two other issues for  resolving`34 skipped, 57 warnings`

**Comment by tinatn29** [comment]
Found the culprits! This [PR](https://github.com/regro/regolith/pull/1190/files) trimmed the whitespaces in two .txt files that were used as the "expected" values in these tests. The assertion errors came from the actual outputs having trailing whitespaces... I'll make sure flake8 ignores .txt files and restore the old versions

**Comment by bobleesj** [comment]
@tinatn29 Great! Yes, you could use `exclude` in one of the `.pre-commit-config.yaml` hooks!

**Comment by sbillinge** [comment]
avoiding txt files is probably ok here.

**Comment by tinatn29** [comment]
Please ignore PR #1197 (closed) and only review #1198

**Comment by bobleesj** [comment]
Closed by https://github.com/regro/regolith/pull/1212
